pkg/api/image.go

```diff
@@ -128,3 +141,44 @@ func (h *Handler) CreateImage(w http.ResponseWriter, r *http.Request) {
 
 	w.Write([]byte(fmt.Sprintf("succeeded created image %s", createdImage.Name())))
 }
+
+// Deletes a block image from this cluster.
+// DELETE
+// /image?name=<imageName>&pool=<imagePool>
```

Comment:
There isn't such thing as an image ID in ceph as far as I know.  All usage of block images has uniquely identified them by image name and pool name.